### DISCUSSION TITLE
(Windows 11) Is there a way to have the shell replace the context menu for everything on the taskbar?

### DISCUSSION THREAD

**Discussion by Flax4** [moudey/Shell#221] [Q&A]
I just found this Shell and I am loving it, I was wondering though (or of it's a problem on my end?) if it's possible to have the Shell replace the context menu for the Win+X menu, right clicking on programs on the taskbar (both pinned and the ones in the system tray) and network/audio + time/date?

On a computer with Windows 10 it seems to replace the Win+X menu but on Windows 11 for me it's not.

**Comment by Eta-Beta** [comment]
Hi, it works for me, I have Win 11 with Explorer Patcher also installed.
![Screenshot 2023-05-12 222624](https://github.com/moudey/Shell/assets/104278263/e3bd031c-e662-4372-832a-3a002e7c4f3e)

**Comment by Assimilator-X** [comment]
You need to switch off the taskbar "optimize taskbar for touch interaction" feature . in taskbar settings.  If you use mouse or the virtual touch-pad it works but for touch it wont.
